features/configuration_loading.feature

```diff
@@ -44,8 +45,9 @@ Feature: Offer different ways how to load configuration
     Then the exit status indicates smells
     And it reports:
       """
-      smelly.rb -- 3 warnings:
+      smelly.rb -- 4 warnings:
         [4, 5]:DuplicateMethodCall: Smelly#m calls @foo.bar 2 times [https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md]
         [4, 5]:DuplicateMethodCall: Smelly#m calls puts @foo.bar 2 times [https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md]
+        [2]:InstanceVariableAssumption: Smelly assumes too much for instance variable @foo [https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md]
         [3]:UncommunicativeMethodName: Smelly#m has the name 'm' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md]
```

Comment:
Can we rather adjust the example code we use here and above to **not** cause the InstanceVariableAssumption warnings?
Did you already rebase against master after #992 has been merged?
Adding a new smell detector should **not** trigger so many changes in unrelated cucumber features. Ideally, it wouldn't cause any.